hypervisor/src/mshv/mod.rs

```diff
@@ -1747,6 +1759,14 @@ impl MshvVcpu {
 
         Ok(())
     }
+
+    #[cfg(target_arch = "aarch64")]
+    fn sys_reg_to_mshv_reg(&self, sys_regs: u32) -> cpu::Result<u32> {
+        match sys_regs {
+            regs::MPIDR_EL1 => Ok(hv_register_name_HV_ARM64_REGISTER_MPIDR_EL1),
```

Comment:
Are there more cases gonna be added later?

Reply:
Yes, there are gonna be few registers in future